### DISCUSSION TITLE
How to run in a docker container?

### DISCUSSION THREAD

**Discussion by decapo01** [giraffe-fsharp/Giraffe#504] [Q&A] [answered]
I'm trying this out and building in docker.  The image builds ok but when I try to run it I'm having trouble connecting to the app 
![image](https://user-images.githubusercontent.com/6868296/152937961-bdb33819-d453-4b0e-985d-aaa529ad9f4a.png)

This is what my docker file looks like
```
# syntax=docker/dockerfile:1
FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
WORKDIR /app

# Copy csproj and restore as distinct layers
COPY . ./
RUN dotnet restore
RUN dotnet build
RUN cd EdbServer && dotnet build
EXPOSE 5000
EXPOSE 5001
ENTRYPOINT ["sh", "entry.sh"]
```
entry.sh looks like this
```
cd EdbServer && dotnet run
```
`EdbServer` is the project with Giraffe in it.
I'm building the docker image and running it with this command ` docker run -p 5000:5000 -p 5001:5001 {dockerImage}`

Also I'm using version 6 alpha 2.

**Comment by 64J0** [comment, marked as answer]
You can find a working example by checking these links:

- Example Dockerfile: https://github.com/giraffe-fsharp/giraffe-website/blob/master/src/Giraffe.Website/Dockerfile
- Check the "How to test" section: https://github.com/giraffe-fsharp/giraffe-website/pull/2
